internal/appsec/remoteconfig.go

```diff
@@ -336,6 +337,16 @@ func (a *appsec) enableRemoteActivation() error {
 	return remoteconfig.RegisterCallback(a.onRemoteActivation)
 }
 
+var blockingCapabilities = [...]remoteconfig.Capability{
```

Comment:
I like the `...` detail ;-)

Reply:
No need to allocate a slice header when all you need is the data vector :)